packages/server/api/src/app/ai/mcp/docs.ts

```diff
@@ -0,0 +1,19 @@
+import { experimental_createMCPClient as createMCPClient } from 'ai';
+import { Experimental_StdioMCPTransport as StdioMCPTransport } from 'ai/mcp-stdio';
+
+const DOCS_MCP_SERVER_PATH =
+  process.env.DOCS_MCP_SERVER_PATH || '/root/.mcp/docs.openops.com';
```

Comment:
use the same way of handling env vars as in the rest of the app

Reply:
Declare it here please https://github.com/openops-cloud/openops/blob/b00419b65e1030664062cbcf80d1f21b3d7f4387/packages/server/shared/src/lib/system/system-prop.ts#L4

and use with system.get

Reply:
done